app/src/main/java/com/github/sdp/mediato/data/UserDatabase.java

```diff
@@ -282,4 +276,23 @@ public static CompletableFuture<Location> getSavedLocation(String username) {
         return future;
     }
 
+    /**
+     * Gets all the nearby users' usernames
+     * @param username of the reference user
+     * @return a completable future with a list of strings containing the usernames
+     */
+    public static CompletableFuture<List<String>> getNearbyUsers(String username) {

```

Comment:
I think it's okay to use a hardcoded radius right now, but it could already be added as a parameter of the function in case the user can pick the search radius himself later. Then the radius could be passed to `findNearbyUsers()`